internal/compiler/generator/cpp.rs

```diff
@@ -3351,6 +3351,10 @@ fn compile_builtin_function_call(
             ctx.generator_state.conditional_includes.cmath.set(true);
             format!("std::atan({}) / {}", a.next().unwrap(), pi_180)
         }
+        BuiltinFunction::ATan2 => {
+            ctx.generator_state.conditional_includes.cmath.set(true);
+            format!("std::atan2({}) / {}", a.next().unwrap(), pi_180)
```

Comment:
```suggestion
            format!("std::atan2({}, {}) / {}", a.next().unwrap(), a.next().unwrap(), pi_180)
```

There are two argument :-)